src/server/db_slice.h

```diff
@@ -409,24 +416,36 @@ class DbSlice {
   // Invalidate all watched keys for given slots. Used on FlushSlots.
   void InvalidateSlotWatches(const SlotSet& slot_ids);
 
-  void PerformDeletion(PrimeIterator del_it, ExpireIterator exp_it, EngineShard* shard,
-                       DbTable* table);
+  void PerformDeletion(PrimeIterator del_it, ExpireIterator exp_it, DbTable* table);
 
   // Send invalidation message to the clients that are tracking the change to a key.
   void SendInvalidationTrackingMessage(std::string_view key);
 
   void CreateDb(DbIndex index);
   size_t EvictObjects(size_t memory_to_free, PrimeIterator it, DbTable* table);
 
-  enum class FindInternalMode {
+  enum class UpdateStatsMode {
     kUpdateReadStats,
     kUpdateMutableStats,
   };
-  ItAndExp FindInternal(const Context& cntx, std::string_view key, FindInternalMode mode);
+
+  enum class LoadExternalMode {
```

Comment:
There is some repetitiveness in the terms: `LoadExternalMode::kLoadExternal`
How about: `LoadMode::kLoadExternal`?
(if you agree, consider also changing above to `StatsMode::kUpdateReadOnly`

Reply:
Sure I agree, though I think LoadExternalMode::kLoad  / LoadExternalMode::kDontLoad it better. Will do the change for both enums